### ISSUE TITLE
Flutter doctor command not working 

### ISSUE THREAD

**Issue by prateekvyas1996** [flutter/flutter#18271]
here is the report
[flutter_01.log](https://github.com/flutter/flutter/files/2081110/flutter_01.log)
[flutter_02.log](https://github.com/flutter/flutter/files/2081111/flutter_02.log)

**Comment by zoechi** [comment]
How did you install?

**Comment by chinmaygarde** [comment]
Seems like the flutter directory is not a git repository? Did you clone the Flutter using Git?

**Comment by zoechi** [comment]
From the website

> Extract the zip-file and place the contained flutter in the desired installation location

> No i just follow the instructions given on website

Doesn't provide any information about where you installed Flutter.
Without information there is no way to help you.
What OS are you using?

**Comment by prateekvyas1996** [comment]
I just extract flutter in c program files

**Comment by zoechi** [comment]
Try a directory instead that doesn't require admin privilegs for writing

**Comment by prateekvyas1996** [comment]
Ok i will try that

**Comment by earlwwhitney** [comment]
> Closing as dup of #16898
> Please add a comment if you don't think this is a duplicate.

I disagree that this is a dup.  #16898 deals with read-only directories.  I followed the directions for flutter installation exactly, even copying the flutter directory to c:\src\ and I have the same problem as the original user.  #16898 was not helpful to me.

**Comment by zoechi** [comment]
This issue is closed since quite some time.
Please create a new issue with fully filled issue template.

**Comment by stuti8P** [comment]
Did anyone have a solution for the issue?

**Comment by kangwang1988** [comment]
@stuti8P 
If you meet this error(reported in flutter_01.log):
```
encountered exception: VersionCheckError: Command exited with code 128: git log -n 1 --pretty=format:%ad --date=iso
Standard error: fatal: not a git repository (or any of the parent directories): .git
```
Please run commands below:
```
cd your-flutter-dir
git log -n 1 --pretty=format:%ad --date=iso
```
your-flutter-dir is supposed to be a git repository, please make sure if it is a git one from your terminal manually.
